python/ray/train/_internal/state/schema.py

```diff
@@ -91,6 +140,9 @@ class TrainRunInfo(BaseModel):
 class TrainRunInfoWithDetails(TrainRunInfo):
     """Metadata for a Ray Train run and information about its workers."""
 
+    workers: List[TrainWorkerInfoWithDetails] = Field(
```

Comment:
Will this override the `workers` field in the parent class (`TrainRunInfo`)?

Reply:
yes, that was my intention.